### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

File: .reek.yml [hunk 1]
```diff
@@ -20,5 +20,4 @@ detectors:
   TooManyStatements:
     max_statements: 10
   TooManyInstanceVariables:
-    exclude:
-      - 'models/**/*.rb'
\ No newline at end of file
+    max_instance_variables: 5
```
Comment (lukasbischof) [inline .reek.yml]: No newline at end of file

File: lib/models/email/content.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  module EMail
+    class Content
+      include ActiveModel::Model
+
+      attr_accessor :account, :amount, :charge_date, :value_date, :memo, :payee_id, :payee_name
+
+      validates_presence_of :account, :amount, :charge_date, :value_date, :memo
+      validates_numericality_of :amount
+      validates_type :charge_date, :date, allow_blank: false
+      validates_type :value_date, :date, allow_blank: false
```
Comment (lukasbischof) [inline lib/models/email/content.rb:15]: Doesn't

```suggestion
      validates_type :value_date, : charge_date, :date, allow_blank: false
```

work?

File: lib/models/ynab/payee.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Models
+  module YNAB
+    class Payee
+      include ActiveModel::Model
+
+      attr_accessor :name, :id
+
+      validate :payee_info?
```
Comment (lukasbischof) [inline lib/models/ynab/payee.rb]: Maybe u could simplify

```suggestion
      validates :name, presence: true, if: id
      validates :id, presence: true, if: :name
```

**Review by lukasbischof — APPROVED** [review]
Looks nice, just added some suggestions. Did not try them, maybe they don't even work 😄

File: lib/models/ynab/payee.rb (hunk 3 above)
Comment (TheFehr) [reply lib/models/ynab/payee.rb]: does this do the same?
I'm confused

File: lib/models/email/content.rb (hunk 2 above)
Comment (TheFehr) [reply lib/models/email/content.rb:15]: might

Comment (TheFehr) [reply lib/models/email/content.rb:15]: nope

**Pull request merged**